header/main.scss

```diff
@@ -1,20 +1,11 @@
 @include nUiCriticalStart('header');
 
-$_n-ui-header-applied: false !default;
-$_n-ui-header-is-silent: nUiHas('header') == false or $_n-ui-header-applied == true;
-$n-header-is-silent: $_n-ui-header-is-silent;
-// NOTE: for now, also needed, until n-header has its own silent mode
-$o-header-is-silent: $_n-ui-header-is-silent;
+$o-header-is-silent: false !default;
 
-@import 'n-header-footer/header_new/header';
-@import "n-header-footer/meganav/meganav";
 @import 'n-header-footer/breadcrumb/breadcrumb';
-@import 'n-header-footer/marketing_promo/marketing-promo';
-
-@if $_n-ui-header-is-silent == false {
-	$_n-ui-header-applied: true !global;
-	$n-header-is-silent: true !global;
-	$o-header-is-silent: true !global;
-}
+@import 'scss/marketing-promo';
+@import 'o-header/main';
```

Comment:
We _can_ import them file by file but because the meganav styles are so small currently it seems too unnecessary to do that now. If, in the future, lots more stuff gets added then we can but for now I don't think we need to. Maybe I can add an issue so that it doesn't get forgotten about?